bin/esi-describe-images

```diff
@@ -57,6 +60,11 @@ class ServiceImageManager(object):
     def __init__(self):
         self._check_environment()
         self._ec2 = boto.connect_ec2_endpoint(os.environ.get("EC2_URL"))
+        token_url = urlparse(os.environ.get("TOKEN_URL"))
+        STSConnection.DefaultRegionEndpoint = token_url.hostname
+        port = token_url.port if token_url.port else 80
+        self._sts = STSConnection(is_secure=False, port=port, path=token_url.path)
+        self._system_accounts = ServiceImageManager._list_system_accounts()
```

Comment:
My concern was for whether it was being used from outside this class or not. I didn't see anything in this PR referencing it.